src/coord/src/catalog/error.rs

```diff
@@ -168,6 +173,18 @@ https://materialize.com/docs/cli#experimental-mode"#
                 r#"Experimental mode is only available on new nodes. For
 more details, see https://materialize.com/docs/cli#experimental-mode"#
             ),
+            ErrorKind::FailedMigration {
+                last_version,
+                cause,
+            } => write!(
+                f,
+                r#"Migration from catalog content version {} failed; underlying error:
```

Comment:
For consistency with the other error messages: "migration from catalog content version {} failed: {}"

(Specifically, lowercase first letter, and the colon already indicates "underlying error." At some point I'd like to make these work more like PostgreSQL with proper error/hint/detail fields, but for now this is the style used by the coord/sql crate.)